modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-0_{context}"]
+= Release Notes for Red Hat {pipelines-title} Technology Preview 1.0
+
+[id="new-features-1-0_{context}"]
+== New features
+* {pipelines-title} 1.0 is available on {product-title} 4.4 and newer versions on channels *ocp-4.4.0* and *preview*.
+* {op-title} 1.0 has been updated to use :
+** Tekton Pipelines 0.11.1
```

Comment:
This has changed AFAIK to a new patch release

Reply:
done